modules/fedimint-ln-common-ng/src/lib.rs

```diff
@@ -0,0 +1,173 @@
+//! # Lightning Module
+//!
+//! This module allows to atomically and trustlessly (in the federated trust
+//! model) interact with the Lightning network through a Lightning gateway.
+//!
+//! ## Attention: only one operation per contract and round
+//! If this module is active the consensus' conflict filter must ensure that at
+//! most one operation (spend, funding) happens per contract per round
```

Comment:
Could you clarify this comment? The consensus logic already provides this assurance right? Or is this something that the module needs to ensure?